gui/browsable/src/RFieldProvider.hxx

```diff
@@ -80,7 +83,7 @@ class RFieldProvider : public RProvider {
       }
 
       template<typename T>
-      void FillHistogram(const RField<T> &field)
+      void FillHistogramImpl(const ROOT::Experimental::RFieldBase &field, ROOT::Experimental::RNTupleView<T, false> &view)
```

Comment:
We should not need to change this, I think.

Reply:
Based on my other comment, I think we are forced to change this... :neutral_face:

Reply:
Ok, let's not forget the follow-up from your comment to remove the tainting of the ROOT namespace